src/main/java/com/snoop/controllers/WorkspaceController.java

```diff
@@ -59,6 +59,10 @@ public String index(Map<String, Object> model) {
 
 		Workspace wkps = new Workspace(new Long(1));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this "Long" constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2129)